assignments/arrays.js

```diff
@@ -63,7 +63,10 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+//Since I know the inventory array will count at 0, asking the console to log the 32nd item in the array will return id 33. 
+// There might've been a more sophisticated way to do this; I will be curious. 
+console.log(inventory[32]);
+console.log(`Car 33 is a 2011 Wrangler Jeep.` );
 
```

Comment:
This is correct, since the 0 position is actually number 1. Number 32 will be 33.